migrations/3_deploy_internalTestsContract.js

```diff
@@ -1,21 +1,23 @@
 /* eslint no-multi-spaces: 0, no-console: 0 */
 
-const Math = artifacts.require('Math')
+const Math2 = artifacts.require('Math2')
 const DutchExchange = artifacts.require('DutchExchange')
 const InternalTests = artifacts.require('InternalTests')
+const Proxy = artifacts.require('Proxy')
+
 
 module.exports = function deploy(deployer) {
-  deployer.link(Math, InternalTests)
-    .then(() => DutchExchange.deployed())
-    .then((dx) => {
+  deployer.link(Math2, InternalTests)
+    .then(() => Proxy.deployed())
+    .then((p) => {
       const initParams = Promise.all([
-        dx.TUL.call(),
-        dx.OWL.call(),
-        dx.auctioneer.call(),
-        dx.ETH.call(),
-        dx.ETHUSDOracle.call(),
-        dx.thresholdNewTokenPair.call(),
-        dx.thresholdNewAuction.call(),
+        DutchExchange.at(p.address).TUL.call(),
```

Comment:
Let's do
```
const dx = DutchExchange.at(p.address)

const initParams = Promise.all([
   dx.TUL.call(),
   ...
```

Reply:
sure...

I feel like a lot of quick changes have been done, just to get it work. Feel free to make any commits to get it cleaner